public/assets/sass/components/_table.scss

```diff
@@ -1,6 +1,219 @@
 .expenses {
     &__table {
-        justify-self: center;
-        width: 90%;
+        margin:0 20px ;
+        @include respond(phone){
+        // justify-self: center;
```

Comment:
Kindly, remove unneeded snippets. If they are saved for future implementation, then add a TODO comment above the statement.